src/main/app/claims/claimModelConverter.ts

```diff
@@ -246,4 +251,11 @@ export class ClaimModelConverter {
       date_created: payment.date_created
     }
   }
+
+  private static convertPhoneNumber(phone: MobilePhone): string {
+    if (phone === undefined) {
+      return undefined;
+    }
+    return StringUtils.trimToUndefined(phone.number)
+  }
```

Comment:
Doesn't cover `null`, which is also falsy. How about just
`return phone ? StringUtils.trimToUndefined(phone.number) : phone`
then
`undefined` => `undefined`
`null` => `null`
`{"number":"something"}` => `"something"`
`{}` => `undefined`
`{"number":"     "}` => `undefined`

Reply:
Done. Unable to send phone back as type clash. so sending undefined back.